CONTRIBUTING.md

```diff
@@ -51,7 +51,13 @@ oauth_token = "my_token"  # Can be left empty if not used
 ```
 
 If you use 2 factor authentication on your Github account, tests that require a login/password authentication will fail.
-You can use `python -m github.tests Issue139.testCompletion --record --auth_with_token` to use the `oauth_token` field specified in `GithubCredentials.py` when recording a unit test interaction. NB that the `password = ""` (empty string is ok) must still be present in `GithubCredentials.py` to run the tests even when the `--auth_with_token` arg is used.
+You can use `python -m github.tests Issue139.testCompletion --record --auth_with_token` to use the `oauth_token` field specified in `GithubCredentials.py` when recording a unit test interaction. Note that the `password = ""` (empty string is ok) must still be present in `GithubCredentials.py` to run the tests even when the `--auth_with_token` arg is used. Note that if you record your test data with `--auth_with_token` then you may need to modify the setup code for your test(s) to authenticate with the token, `self.g = github.Github(self.oauth_token)`.
+
```

Comment:
Update previous line to indicate how a test can authenticate with token (even when `--auth_with_token` not passed on command line). I still wonder if the Framework should be modified to make this unnecessary?

Reply:
> Also note that if you record your test data with `--auth_with_token` then you should modify the setup code for your Framework.TestCase to set `self.TokenAuthMode = True` before calling `Framework.TestCase.setUp(self)`.

I am not sure if this is necessary. If I wanna authenciate with token, shouldn't I always pass the `--auth_with_token` command line parameter?

Also, I am thinking to disable the password feature, since we don't want to accidentally commit 
 password to git and some users might be concerned about leaving their password in plaintext. Always use a personal access token, with a set of permissions defined by the user. What's your thoughts on this?